src/Expression.js

```diff
@@ -107,6 +107,9 @@ class Expression {
    */
   evaluate(expression, context) {
     try {
+      if (expression.match(/include\(\'.*\'\)/)) {
```

Comment:
cose evaluating for built-in functions happens before it comes to _includeSource function. So, we can't fix include() built-in function case.